pkg/kubectl/cmd/autoscale/autoscale.go

```diff
@@ -133,7 +133,11 @@ func (o *AutoscaleOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args
 	o.dryRun = cmdutil.GetFlagBool(cmd, "dry-run")
 	o.createAnnotation = cmdutil.GetFlagBool(cmd, cmdutil.ApplyAnnotationsFlag)
 	o.builder = f.NewBuilder()
-	o.canBeAutoscaled = polymorphichelpers.CanBeAutoscaledFn
+	d, err := f.ToDiscoveryClient()
```

Comment:
please try to avoid one-letter variables.  They're unclear, hard to refactor, and easy to lose track of.

Reply:
Done. Thanks.